homeassistant/components/netatmo/light.py

```diff
@@ -134,3 +150,64 @@ async def async_turn_off(self, **kwargs: Any) -> None:
     def async_update_callback(self) -> None:
         """Update the entity's state."""
         self._is_on = bool(self._camera.floodlight == "on")
+
+
+class NetatmoLight(NetatmoBase, LightEntity):
+    """Representation of a dimmable light by Legrand/BTicino."""
+
+    def __init__(
+        self,
+        netatmo_device: NetatmoDevice,
+    ) -> None:
+        """Initialize a Netatmo light."""
+        LightEntity.__init__(self)
+        super().__init__(netatmo_device.data_handler)
+
+        self._dimmer = cast(NaModules.NLFN, netatmo_device.device)
+        self._id = self._dimmer.entity_id
+        self._home_id = self._dimmer.home.entity_id
+        self._device_name = self._dimmer.name
+        self._attr_name = f"{self._device_name}"
```

Comment:
```suggestion
        self._attr_name = self._dimmer.name
```
No special use for _device_name so you can skip it

Reply:
It is used in the base class though. E.g. https://github.com/home-assistant/core/blob/add_light/homeassistant/components/netatmo/netatmo_entity_base.py#L96